testutil/integration/evmos/network/amounts.go

```diff
@@ -0,0 +1,76 @@
+// Copyright Tharsis Labs Ltd.(Evmos)
+// SPDX-License-Identifier:ENCL-1.0(https://github.com/evmos/evmos/blob/main/LICENSE)
+package network
+
+import (
+	"math/big"
+
+	"cosmossdk.io/math"
+	"github.com/evmos/evmos/v20/types"
+	evmtypes "github.com/evmos/evmos/v20/x/evm/types"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+)
+
+type InitialAmounts struct {
+	Base math.Int
+	Evm  math.Int
+}
+
+func DefaultInitialAmounts() InitialAmounts {
+	baseCoinInfo := evmtypes.ChainsCoinInfo[defaultChain]
+
+	return InitialAmounts{
+		Base: GetInitialAmount(baseCoinInfo.Decimals),
+		Evm:  GetInitialAmount(baseCoinInfo.Decimals),
+	}
+}
+
+func DefaultInitialBondedAmount() math.Int {
+	baseCoinInfo := evmtypes.ChainsCoinInfo[defaultChain]
+
+	return GetInitialBondedAmount(baseCoinInfo.Decimals)
+}
```

Comment:
I think these `Default...` functions can be removed, since they are only called in the `DefaultConfig` where the `InitialAmount(...)` function can just be called with the corresponding denom - would be clearer to me TBH, since this currently feels like a lot of spaghetti code